front/app/containers/AccessibilityStatement/index.tsx

```diff
@@ -121,9 +121,7 @@ const CookiePolicy = memo((props: WrappedComponentProps) => {
                 <ul>
                   <li>
                     {formatMessage(messages.email)}{' '}
-                    <a href="mailto:support@citizenlab.co">
-                      support@citizenlab.co
-                    </a>
+                    <a href="mailto:support@govocal.co">support@govocal.co</a>
                   </li>
```

Comment:
@sebastienhoorens Same here - emails will still be `.co` right?

Reply:
I think we are going with .com also for the email domain (@kogre could you perhaps confirm?)

Reply:
I'm still not sure what the timing of that will be. Maybe it's safer to not change it yet, as we'll have redirects in any case.